fix repository URLs in packages

zeke opened the pull request:
packages → modules

<img width="402" alt="screen shot 2017-07-03 at 11 23 55 am" src="https://user-images.githubusercontent.com/2289/27804151-3780379a-5fe2-11e7-8aed-4f35598c2a51.png">

cc @broccolini

broccolini commented:
Thanks @zeke - that's actually only one of the packages and not the actual `primer-css` package. And all of the packages should be updated to correct this url! Since we'll need to cut a new release I've pulled your pr into a [`dev` branch](https://github.com/primer/primer-css/pull/243) as we may want to include a few other additions before cutting a new release. Thanks for spotting this one though!

zeke commented:
> that's actually only one of the packages

oops thought I did a global search and replace 🙈 

Feel free to close this out then 👍

zeke commented:
> thought I did a global search and replace

Figured it out. I switched from Atom to VSCode which doesn't save search-and-replace changes automatically by default.

I went ahead and updated the PR to fix all URLs just in case you still need it.

broccolini commented:
@shawnbot fyi I pulled this into my pr https://github.com/primer/primer-css/pull/243
